### ISSUE TITLE
davinci-helper

### ISSUE THREAD

**Issue by Malix-Labs** [ublue-os/config#339]
About

https://github.com/ublue-os/config/blob/74905f53c65977066ab7caed15aded3048e685c6/build/ublue-os-just/30-distrobox.just#L83-L84

Have you considered using https://github.com/H3rz3n/davinci-helper (https://copr.fedorainfracloud.org/coprs/herzen/davinci-helper/) instead ?

**Comment by Zeglius** [comment]
So, trying to set it up, but is presenting some issues, making it inoperable inside containers at the moment:
- https://github.com/H3rz3n/davinci-helper/issues/15
- https://github.com/H3rz3n/davinci-helper/issues/16
Those are the mayor blockers at the time.

On the UX side, I feel is *too complicated*, in the sense that:
![Image](https://github.com/user-attachments/assets/8db915d4-f2a3-47a2-94a5-8522452e14b6)
These are tabs on the app, each with an step of the process to install davinci, which need to be triggered manually with the start button 
![Image](https://github.com/user-attachments/assets/9b1328be-67be-4f60-bebf-55afc0620f5e)

I (personally) feel is something unnecessary and that could be automated/omitted.

TL;DR: This feels a great tool, but I dont think it fits our needs for 'simple to use' scripts.
Of course, this is my take, I ask for others opinions on whenever we should use this instead of davincibox, specially when comes to the UX side.

Though, must remind that these issues linked above prevent us of event setting it up, need to be taken care of.

**Comment by Malix-Labs** [comment]
Surely a distrobox (or even better, Flatpak) would be better

## Related

- https://forum.blackmagicdesign.com/viewtopic.php?t=86919&p=483611
- https://discourse.flathub.org/t/davinci-resolve-flatpak-request/842
- https://github.com/pobthebuilder/resolve-flatpak
